src/test/java/seedu/sudohr/logic/parser/department/FindDepartmentCommandParserTest.java

```diff
@@ -0,0 +1,35 @@
+package seedu.sudohr.logic.parser.department;
+
+import static seedu.sudohr.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.sudohr.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.sudohr.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.sudohr.logic.commands.department.FindDepartmentCommand;
+import seedu.sudohr.model.department.DepartmentNameContainsKeywordsPredicate;
+
+public class FindDepartmentCommandParserTest {
+
+    private FindDepartmentCommandParser parser = new FindDepartmentCommandParser();
+
+    @Test
```

Comment:
I recommend having an additional test cases that include PREFIXES e.g in the other test file like DeleteCommandParserTest, it takes into account cases where additional fields (prefix fields) provided. 

Not sure if the intended behaviour of the error message is NAME not formatted correctly or invalid command usage. I think the former aligns with most but do check it out! Thanks

Reply:
I realised that the reason why the cases where additional fields are provided fails for DeleteCommandParser is because it tries to cast the entire string after the eid prefix into employee id. I dont think any of the parser handles the case where we input redundant arguments using prefixes. 
<img width="543" alt="Screenshot 2023-03-21 at 7 26 17 PM" src="https://user-images.githubusercontent.com/59330153/226593221-dc2b2fea-b1ac-4af5-ae10-224a8a6f1a18.png">

Reply:
But yea, you are right that checks regarding valid inputs for referencing existing departments should be added, but since this is a find command using keywords (for which none of the departments might match), I thought that the check was not necessary.

Reply:
also for DeleteCommandParser, that is the expected behavior right? The handling of redundant argument i believe falls under the regex for each prefix and tokenizer.

so eg. name fields should not allow '/' something like:
id/77 n/andre r/abc e/... a/.. p/...
would fail because name field is andre r/abc, '/' is illegal. This is just a simple example to check against expected behavior.